internal/array/contains.go

```diff
@@ -40,7 +40,7 @@ func ContainsStrCaseInsensitive(array []string, expected string) bool {
 
 func ContainsPartialStr(array []string, expected string) bool {
 	for _, value := range array {
-		if strings.Contains(value, expected) {
+		if strings.Contains(expected, value) {
```

Comment:
this is a big change for this function, It was never used but also, there is no unit tests so it is hard to tell if this is good or bad.